openquakeplatform/openquakeplatform/irv/static/irv/js/irv_viewer.js

```diff
@@ -102,13 +127,13 @@ function scaleTheData() {
     }
 
     // Put values back into the layerAttributes obj
-    for (var key in indicatorsToBeScaled) {
-        for (var i = 0; i < layerAttributes.features.length; i++) {
-            for (var layerAttributesKey in layerAttributes.features[i].properties) {
-                if (key === layerAttributesKey) {
-                    for(var indicatorKey in indicatorsToBeScaled[key]) {
-                        if (layerAttributes.features[i].properties[selectedRegion] == indicatorKey) {
-                            layerAttributes.features[i].properties[layerAttributesKey] = indicatorsToBeScaled[key][indicatorKey];
+    for (var ind2 in indicatorsToBeScaled) {
+        for (var i3 = 0; i3 < layerAttributes.features.length; i3++) {
+            for (var layerAttributesKey in layerAttributes.features[i3].properties) {
+                if (ind2 === layerAttributesKey) {
+                    for(var indicatorKey in indicatorsToBeScaled[ind2]) {
+                        if (layerAttributes.features[i3].properties[selectedRegion] == indicatorKey) {
+                            layerAttributes.features[i3].properties[layerAttributesKey] = indicatorsToBeScaled[ind2][indicatorKey];
```

Comment:
`ind2` and `i3` have a scope just inside the loop, why use different names compared with previous loop ?